app/src/main/AndroidManifest.xml

```diff
@@ -32,7 +30,9 @@
         <activity
             android:name="io.left.meshim.activities.MainActivity"
             android:label="@string/app_name" />
-        <activity android:name="io.left.meshim.activities.ChatActivity" />
+        <activity android:name="io.left.meshim.activities.ChatActivity">
```

Comment:
This whole change shouldn't be here - you have a redundant closing XML tag. Basically there shouldn't be any changes to this file.